cmd/airflow_test.go

```diff
@@ -139,28 +149,38 @@ func (s *AirflowSuite) TestNewAirflowInitCmd() {
 }
 
 func (s *AirflowSuite) TestNewAirflowRunCmd() {
-	cmd := newAirflowRunCmd()
-	s.Nil(cmd.PersistentPreRunE(new(cobra.Command), []string{}))
+	cmd := newAirflowInitCmd()
+	cmd.RunE(new(cobra.Command), []string{})
```

Comment:
@pritt20 sorry I may have missed the explanation earlier but why did we end up adding these two lines to run `init` in each of these tests? Why didn't we need this before?

Reply:
Hey @schnie ,

Previously we were checking for `PersistentPreRunE` hook in these tests for each command but with this change we have moved `PersistentPreRunE` to root dev command and hence its no longer required for sub-commands and therefore these tests were failing.

I refactored these tests to check for `PreRunE` hook instead but in order to do that we had to initialise the project first and hence an extra step  to use `newAirflowInitCmd` to invoke the `PreRunE` hook.